src/pages/OnboardingAccounting/types.ts

```diff
@@ -0,0 +1,12 @@
+import type {StackScreenProps} from '@react-navigation/stack';
+import type {OnboardingModalNavigatorParamList} from '@libs/Navigation/types';
+import type SCREENS from '@src/SCREENS';
+
+type OnboardingAccountingProps = Record<string, unknown> & StackScreenProps<OnboardingModalNavigatorParamList, typeof SCREENS.ONBOARDING.ACCOUNTING>;
```

Comment:
What's is this type `Record<string, unknown>` for?

Reply:
removed